build.gradle

```diff
@@ -40,6 +40,10 @@ File getEnvironmentScript()
 apply from: environmentScript
 apply from: "${buildScriptDirPath}/configBuildScript.gradle"
 
+def versions = [
```

Comment:
Could you do us a favor and group-lock other dependencies while you're at it? (e.g. spring, parseq, guice)

Reply:
Absolutely, done!